src/containers/BooksList.js

```diff
@@ -1,27 +1,55 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import Table from 'react-bootstrap/Table';
 import Book from '../components/Book';
+import { removeBook } from '../actions/index';
 
-const BooksList = books => {
-  // eslint-disable-next-line max-len
-  const BooksList = books.books.books.map(b => <Book key={b.bookId} bookId={b.id} title={b.title} category={b.category} />);
-  return (
-    <Table striped bordered hover>
-      <thead>
-        <tr>
-          <th>BookID</th>
-          <th>Title</th>
-          <th>Category</th>
-        </tr>
-      </thead>
-      {BooksList}
-    </Table>
-  );
+class BooksList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleRemove = this.handleRemove.bind(this);
+  }
+
+  handleRemove(id) {
+    // eslint-disable-next-line react/destructuring-assignment
+    this.props.removeBook(id);
+  }
+
+  render() {
+    const { books } = this.props;
+    const BooksList = books.books.map(
+      // eslint-disable-next-line max-len
+      b => <Book key={b.id} Id={b.id} title={b.title} cat={b.category} Click={() => this.handleRemove(b.id)} />,
```

Comment:
- You could have this statement in multiple lines to solve this , no need to disable it